src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.jsx

```diff
@@ -54,6 +56,61 @@ export default class SentryAppDetailsModal extends AsyncComponent {
     });
   }
 
+  get permissions() {
+    return new ConsolidatedScopes(this.props.sentryApp.scopes).toPermissions();
+  }
+
+  renderPermissions() {
+    const permissions = this.permissions;
+    const {organization, sentryApp} = this.props;
+    return (
+      <React.Fragment>
+        <p>
+          {t('Install on your ')}
+          <strong>{organization.slug}</strong>
+          {t(' organization with the following permissions:')}
+        </p>
+        <Panel>
+          {permissions.read.length > 0 && (
+            <PanelItem key="read">
+              <Text>
+                <strong>{t('Read')}</strong>
+                {t(` access to ${permissions.read.join(', ')}`)}
```

Comment:
@MeredithAnya When you have templated text for translations, you can't really take this approach. Otherwise, you'd need a different translation key for every possible combination of permissions. Here's the better way of doing  this:
```
t(' access to %s', permissions.read.join(', '))
```
This way, we don't need different translations for every combination of permissions.